### PR TITLE
release 2.1.0

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

**Comment by garyverhaegen-da** [comment]
Per release planning, current RC is `2.1.0-snapshot.20220404.9660.0.4b71431b`, which is built on commit 4b71431b. The [checklist](https://docs.google.com/document/d/1RY2Qe9GwAUiiSJmq1lTzy6wu1N2ZSEILQ68M9n8CHgg/edit#heading=h.d7hqcxt8hr80) indicates all tests have passed.


Reminder: this is a stable release and thus requires a team lead's approval before merging.

**Review by ray-roestenburg-da — APPROVED** [review]

**Pull request merged**
